pysmt/optimization/goal.py

```diff
@@ -77,6 +93,9 @@ def __init__(self, formula):
         """
         self.formula = formula
 
+    def opt(self):
+        return MaximizationGoal
```

Comment:
Are we returning a class here? Is this the pythonic way to do it? *(just curious)*